misc/python/materialize/mzbuild.py

```diff
@@ -465,13 +472,16 @@ def sync_description(self) -> None:
                 f"--file={readme_path}",
                 *([f"--config={docker_config}/config.json"] if docker_config else []),
                 *([f"--short={self.description}"] if self.description else []),
-                f"materialize/{self.name}",
+                self.docker_name(),
             ]
         )
 
-    def docker_name(self, tag: str) -> str:
+    def docker_name(self, tag: str | None = None) -> str:
         """Return the name of the image on Docker Hub at the given tag."""
-        return f"materialize/{self.name}:{tag}"
+        name = f"{self.rd.image_registry}/{self.rd.image_prefix}{self.name}"
+        if tag:
```

Comment:
```suggestion
        if tag is not None:
```
But an empty `""` tag probably doesn't make much sense anyway.